DEVELOPING_TUTORIALS.md

```diff
@@ -468,40 +465,20 @@ return {
 ![screenshot](public/markdown_error_logdesc_log.png)
 
 
-### Manage your tutorial's metadata and routing
+### Manage your tutorial's metadata
 
 There are a few administrative steps you need to take to ensure that all of your lessons appear on our site, along with a landing page and resources page.
 
-Note that you won't be able to preview your lessons until your routing is set up, so you'll need to refer to this section regularly during your development.
-
 #### Add your tutorial to `static/tutorials.json`
 
 In `static/tutorials.json`, add a new key for your tutorial and fill in the appropriate values, as in the example below.
 
 ```json
 "0005": {
```

Comment:
Are you missing the `redirects` in this example? Let's include it with whatever name we choose and a blank array.